fixtures/vitest-pool-workers-examples/external-package-resolution/src/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
 	"extends": "../../tsconfig.workerd.json",
-	"include": ["./**/*.ts"]
+	"include": ["./**/*.ts"],
+	"compilerOptions": {
+		"skipLibCheck": true
```

Comment:
Let's put a comment here explaining why we are not checking libs - i.e. that the toucan-js references a different workers-types version which breaks overall type checking.